docs/notebooks/tutorials/EffectiveRipple.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<p>do we need to plot the force error?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/1003/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
No, is this script fine: https://github.com/PlasmaControl/DESC/blob/ku/fourier_bounce_neo/docs/notebooks/tutorials/EffectiveRipple.ipynb?